frappe/config/__init__.py

```diff
@@ -1,11 +1,5 @@
 import frappe
-from frappe import _
-from frappe.desk.moduleview import (
-	config_exists,
-	get_data,
```

Comment:
I had a brief chat about removing imports from frappe, the main problem was someone could easily be using this imported value in their app, even though it's not being used in this file itself.  

`frappe.config.get_data`

Reply:
If we need to preserve, we can mark(# noqa) selected import and leave as is.

Reply:
yep, some probably used ones can be marked noqa and left as is like `__init__.py` file ones.

Or we can exclude __init__.py files totally from this check too.